assets/sass/protocol/includes/themes/_index.scss

```diff
@@ -0,0 +1,47 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// This file contains all the theme variables for both the Firefox and Mozilla
+// themes. Mixins and functions can draw from these variable maps, swapping to
+// a different map based on the value of the global $brand-theme variable.
+
+@import 'mozilla';
+@import 'firefox';
+
+:root {
+    --v-grid-xs: #{$layout-md};
+    --h-grid-xs: #{$layout-xs};
+
+    // sm is the same as xs
+    --v-grid-sm: #{$layout-md};
+    --h-grid-sm: #{$layout-xs};
+
+    // md
+    --v-grid-md: #{$layout-md};
+    --h-grid-md: #{$layout-lg};
+
+    // lg is the same as medium
+    --v-grid-lg: #{$layout-md};
+    --h-grid-lg: #{$layout-lg};
+
+    // xl
+    --v-grid-xl: #{$layout-xl};
+    --h-grid-xl: 80px;
+
+    @if $brand-theme == 'mozilla' {
+        @extend %mozilla-theme;
+        @extend %mozilla-type-scale;
+
+        @if $type-scale == 'condensed' {
```

Comment:
👏  I was thinking all the custom properties would be set on the `:root` inside the individual theme files but that would make the type scale logic trickier. This is an effective, readable way of breaking them up.